src/index.js

```diff
@@ -90,7 +90,7 @@ function setScope(options) {
     if (toConsole.length) cLogger.error(...toConsole);
   };
 
-  window.onerror = (...args) => {
+  global.onerror = (...args) => {
```

Comment:
This should really really use `addEventListener` instead of overriding `onerror`, because this destroys whatever `onerror` was before.